---
layout: page
title: Expertise offered by the Sheffield Bioinformatics Core
excerpt: "How to work with the SBC..."
---

As part of our *triage* process we will endeavour your query to the most appropriate resource, which may be the faculty members that make up the [Sheffield Bioinformatics Hub](http://bioinformatics.group.shef.ac.uk/) or the [Research Software Engineering group](http://rse.shef.ac.uk/) at Sheffield.

The SBC has expertise in the following areas, and software packages:-

## High-Throughput Sequence Analysis

- BWA, Bowtie and other aligners for all NGS platforms and applications
- Tophat, Cufflinks, DESeq2 and salmon (among others) for RNA-Seq analysis
- GATK, VarScan and MuTect for somatic and germline variant calling
- MACS, and other peak callers for ChIP-Seq analysis
- Downstream analysis of gene lists to identify over-represented pathways / ontologies using [GSEA](http://software.broadinstitute.org/gsea/index.jsp), goseq, or other online tools

## Microarray Analysis

- Experience with data from Affymetrix and Illumina platforms
- Varying data types, including expression, methylation (450K), SNP and copy-number
- Bioconductor/R


## Further Expertise

- Custom pipeline creation
- Custom visualisation of high-throughput data
- Creating interactive web applications wih [shiny](https://shiny.rstudio.com/)

As well as these analysis services, we also provide training courses and bespoke 1-to-1 training for researchers


