---
layout: page
title: Mark Dunning
excerpt: "More about Mark..."
---

![mark](../images/IMG_2921_resize.jpg)

Mark joined The University of Sheffield in October 2017 to establish the Bioinformatics Core

- [twitter](https://twitter.com/DrMarkDunning)
- [github](https://github.com/markdunning)
- [orcid](https://orcid.org/0000-0002-8853-9435)

## Experience

2015 – 2017: Bioinformatics Training Co-ordinator, Cancer Research Uk, Cambridge Institute

2009 – 2015: Bioinformatics Analyst / Senior Bioinformatics Analyst, Cancer Research Uk Cambridge Institute

2005 – 2009: PhD (Oncology) University of Cambridge

2004 – 2005: Msc (Data Analysis, Networks and Nonlinear Dynamics) University of York

1999 – 2004: Bsc (Mathematics and Computer Science) University of York

 
## Biography

I obtained my PhD in the Statistics and Computational Biology group of [Simon Tavare](https://www.compbio.group.cam.ac.uk/) at The University of Cambridge. As part of my [thesis](https://www.repository.cam.ac.uk/handle/1810/218542) I developed open-source software for the analysis of Illumina microarray data, which is available through the [Bioconductor project](http://www.bioconductor.org). I joined the [Bioinformatics Core](https://www.cruk.cam.ac.uk/core-facilities/bioinformatics-core) at [Cancer Research Uk Cambridge Institute](https://www.cruk.cam.ac.uk/) and played a key role in the analysis of gene expression profiles as part of the [METABRIC](https://www.ncbi.nlm.nih.gov/pubmed/22522925) project, which identified and described new subtypes of breast cancer. I also participated in the pilot phases of the International Cancer Genome Consortium (ICGC) project by developing computational pipelines to process the whole-genome sequencing data from Oesophageal cancer patients. During my time in the Bioinformatics Core I also developed a passion for teaching and commenced a role dedicated to organising and delivering [Bioinformatics training courses](https://github.com/bioinformatics-core-shared-training), with the aim of empowering wet-lab scientists to begin to explore data for themselves and foster more-productive collaborations with Bioinformaticians.

I have a strong commitment to reproducible research and making my research outputs available to other researchers, and indeed members of the public who may have funded the research in the first place. For instance, I recently developed and deployed a [Shiny](https://shiny.rstudio.com/) application that allows interested parties to query various [prostate cancer datasets](https://bioinformatics.cruk.cam.ac.uk/apps/camcAPP/). In keeping with my open access principles, the code underlying the application is available via [github](https://github.com/crukci-bioinformatics/camcAPP) and utilises data sets that can be downloaded from [Bioconductor](https://bioconductor.org/packages/release/BiocViews.html#___ProstateCancerData). I have also recently investigated technologies such as Galaxy and [Docker](https://www.docker.com) to ease the deployment of software and facilitate reproducible research.

## Publications

For my updated and full list of publications, please see my ORCid entry:-

[https://orcid.org/0000-0002-8853-9435](https://orcid.org/0000-0002-8853-9435)

Selected Publications

1. Dunning, M.J, Vowler, S.L., Lalonde, E., Ross-Adams. H,. Boutros, P., Mills, I.G., Lynch, A.G, and Lamb A.D., “Mining Human Prostate Cancer Datasets: The camcAPP Shiny App“ EBio Medicine 17, (2017)

2. Ross-Adams, H., Lamb, A.D., Dunning, M.J., Halim, S., Lindberg, J., Massie, C.M., Egevad, L.A., Russell, R., Ramos-Montoya, A., Vowler, S.L., Sharma, N.L., Kay, J., Whitaker, H., Clark, J., Hurst, R., Gnanapragasam, V.J, Shah, N.C, Warren, A.Y., Cooper, C.S., Lynch, A.G., Stark, R., Mills, I.G., Gronberg, H. and Neal, D.E., “Integration of copy number and transcriptomics provides risk stratification in prostate cancer: A discovery and validation cohort study“ EBio Medicine 2(9):1133-1144, (2015).

3. Weaver, J.M.J., Shannon, N., Lynch, A.G., Ross-Innes, C.S., Forshew, T., Barbera, M., Ong, C.J., Lao-Sirieix, P., Dunning, M.J., Smith, L., Smith, M., Carvalho, B., O’Donovan. M., Underwood, T., Murtaza, M., May, A.P., Grehan, N., Hardwick, R., Davies, J., Olomi, A., Aparicio, S., Rozenfeld, N., Eldridge, M., Caldas, C., Edwards, P.A.W., Tavare, S. and Fitzgerald, R.C. “Ordering of mutations in preinvasive disease stages of esophageal carcinogenesis” Nature Genetics epub ahead of print (2014) ?

4. Dawson, S.J., Tsui, D.W.Y, Murtaza, M., Biggs, H., Rueda, O.M., Chin, S.F., Dunning, M.J., Gale, D., Forshew, T., Mahler-Araujo, B., Rajan, S., Humphray, S., Becq, J., Halsall, D., Wallis, M., Bentley, D., Caldas, C. and Rosenfeld, N., “Analysis of Circulating Tumour DNA to Monitor Metastatic Breast Cancer” The New England Journal of Medicine 368:1199-1209, 2013 ?

5. Curtis, C., Shah S., Chin, S.F., Turashvili, G., Rueda, O.M., Dunning, M.J., Speed, D., Lynch, A.G., Samarajiwa, S., Y, Yuan., METABRIC Group et al “The genomic and transcriptomic architecture of 2,000 breast tumours reveals novel subgroups.” Nature, 481(7381):389-93,2012 ?

6. Lynch, A.G., Chin, S.F., Dunning, M.J., Caldas, C., Tavare, S., and Curtis, C., “Calling sample mix-ups in cancer population studies” PLoS One, 7(8) e41815, 2012 ?

7. Ritchie, M.E., Dunning, M.J., Smith, M.L., Shi, W, and Lynch, A.G., “BeadArray expression analysis using bioconductor” PLoS Computational Biology 7(12) e1002276, 2011 ?

8. Dunning, M.J., Curtis, C., Barbosa-Morais, N.L., Caldas, C., Tavare, S. and Lynch, A.G., “The importance platform annotation in interpreting microarray data” Lancet Oncology 11(8):717, 2010 ?

9. Barbosa-Morais, N.L, Dunning, M.J., Samarajiwa, S.A., Darot, J.F.J., Ritchie, M.E., Lynch, A.G. and TavarE, S. “A re-annotation pipeline for Illumina BeadArrays: improving the interpretation of gene expression data” Nucleic Acids Research 38(3):e17, 2009 ?

10. Dunning, M.J., Barbosa-Morais, N.L, Lynch,. A.G., Tavare, S. and Ritchie, M.E. “Statistical issues in the analysis of Illumina data” BMC Bioinformatics 9:85, 2008 ?

11. Dunning, M.J. Smith, M.L., Ritchie, M.E. and Tavare, S. “beadarray: R classes and methods for Illumina bead-based data” Bioinformatics 23(16):2183-4 
