---
layout: page
title: Matthew Parker
excerpt: "More about Matt..."
---

![matt](../images/matt.jpg)

I joined University of Sheffield Bioinformatics Core in May 2018 to provide support for the Biomedical Research Centre (BRC) 
and other university projects in addition to strengthening the clinical bioinformatics services. 

- [twitter](https://twitter.com/bioinfomatt)
- [github](https://github.com/mattdmem)
- [google scholar](https://scholar.google.co.uk/citations?user=izGXAUMAAAAJ)

## Experience

2016 - 2018: Lead Bioinformatician, Sheffield Diagnostics Genetics Service, Sheffield Children's Hospital, Sheffield, UK

2014 - 2016: Lead Cancer Bioinformatician, Genomics England, London, UK

2013 – 2014: Bioinformatics Associate Research Scientist, Clinical Genome Project, St Jude Children's Research Hospital, Memphis TN, UK

2010 – 2013: Postdoctoral Scientist, The Pediatric Cancer Genome Project, St Jude Children's Research Hospital, Memphis TN, UK

2006 – 2010: PhD (Oncology) Institute of Cancer Research, London, UK

2002 – 2006: Bsc (With Honours in Experimental Pathology) University of Edinburgh, Scotland, UK

 
## Biography

I obtained my PhD in Molecular Biology/Cancer Biology at The Institute of Cancer Research in Colin Cooper's group where I developed a keen interest in the somatic changes which lead to the development of cancer. I pursued this interest at St Jude Children's Hospital in Memphis, TN, USA where I completed my postdoctoral training as part of the Pediatric Cancer Genome Project (PCGP) team. During this time I was responsible for the analysis of whole genome sequencing data in addition to transcriptomic data derived from childhood brain tumours. These analyses lead to discoveries which increased our understanding of these tumours. In Ependymoma I discovered a highly recurrent fusion gene, C11orf95-RELA.

I applied the skills I had learnt in bioinformatics and genome analysis to a clinical setting. I was lead analyst on the clinical programme for genomics at St Jude, applying whole genome, exome and transcritome sequencing as a diagnostic tool. I took part in preparations to gain CAP-CLIA certification. On return to the UK I took a role as Lead Cancer Analyst on the 100,000 genomes project, applying my bioinformatics skills again in a clinically focused setting. Subsequently I joined Sheffield Diagnostics Genetics Service to provide bioinformatics suppot to clinical testing. I led a small team of bioinformaticians, building, deploying and supporting bioinformatics pipelines and applications. I successfully supported training for the clinical scientist training programme (STP). I brought the bioinformatics team and processes up to ISO-15189 standards to help attain UKAS accreditation for the laboratory as a whole. During this time I also sought Clinical Scientist Equivalence from the Academy of Healthcare Sciences. I obtained this in early 2018 and registered as a Clinical Scientist with the HCPC (Registration Number CS20252).

## Publications

For my updated and full list of publications, please see my google scholar page:-

[https://scholar.google.co.uk/citations?user=izGXAUMAAAAJ](https://scholar.google.co.uk/citations?user=izGXAUMAAAAJ)

### Selected Publications

1. Parker M, Mohankumar KM, Punchihewa C, Weinlich R, Dalton JD, Li Y, Lee
R, Tatevossian RG, Phoenix TN, Thiruvenkatam R, White E, Tang B, Orisme
W, Gupta K, Rusch M, Chen X, Li Y, Nagahawhatte P, Hedlund E, Finkelstein
D, Wu G, Shurtleff S, Easton J, Boggs K, Yergeau D, Vadodaria B, Mulder
HL, Becksfort J, Gupta P, Huether R, Ma J, Song G, Gajjar A, Merchant T,
Boop F, Smith AA, Ding L, Lu C, Ochoa K, Zhao D, Fulton RS, Fulton LL,
Mardis ER, Wilson RK, Downing JR, Green DR, Zhang J, Ellison DW,
Gilbertson RJ: C11orf95-RELA fusions drive oncogenic NF-kappaB
signalling in ependymoma. Nature 2014, 506(7489):451-455.

2. Robinson G, Parker M, Kranenburg TA, Lu C, Chen X, Ding L, Phoenix TN,
Hedlund E, Wei L, Zhu X, Chalhoub N, Baker SJ, Huether R, Kriwacki R,
Curley N, Thiruvenkatam R, Wang J, Wu G, Rusch M, Hong X, Becksfort J,
Gupta P, Ma J, Easton J, Vadodaria B, Onar-Thomas A, Lin T, Li S, Pounds
S, Paugh S, Zhao D, Kawauchi D, Roussel MF, Finkelstein D, Ellison DW,
Lau CC, Bouffet E, Hassall T, Gururangan S, Cohn R, Fulton RS, Fulton LL,
Dooling DJ, Ochoa K, Gajjar A, Mardis ER, Wilson RK, Downing JR, Zhang J,
Gilbertson RJ: Novel mutations target distinct subgroups of
medulloblastoma. Nature 2012, 488(7409):43-48.

3. Parker M, Chen X, Bahrami A, Dalton J, Rusch M, Wu G, Easton J, Cheung
NK, Dyer M, Mardis ER, Wilson RK, Mullighan C, Gilbertson R, Baker SJ,
Zambetti G, Ellison DW, Downing JR, Zhang J, Pediatric Cancer Genome P:
Assessing telomeric DNA content in pediatric cancers using whole-
genome sequencing data. Genome Biol 2012, 13(12):R113.

4. Cheung NK, Zhang J, Lu C, Parker M, Bahrami A, Tickoo SK, Heguy A,
Pappo AS, Federico S, Dalton J, Cheung IY, Ding L, Fulton R, Wang J, Chen
X, Becksfort J, Wu J, Billups CA, Ellison D, Mardis ER, Wilson RK, Downing
JR, Dyer MA, St Jude Children&#39;s Research Hospital-Washington University
Pediatric Cancer Genome P: Association of age at diagnosis and genetic
mutations in patients with neuroblastoma. JAMA 2012, 307(10):1062-
1071.
