---
layout: page
title: Matthew Parker
excerpt: "More about Mark..."
---

![mark](../images/IMG_2921_resize.jpg)

I joined University of Sheffield Bioinformatics Core in May 2018 to provide support for the Biomedical Research Centre (BRC) 
and other university projects in addition to strengthening the clinical bioinformatics services. 

- [twitter](https://twitter.com/bioinfomatt)
- [github](https://github.com/mattdmem)
- [google scholar](https://scholar.google.co.uk/citations?user=izGXAUMAAAAJ)

## Experience

2016 - 2018: Lead Bioinformatician, Sheffield Diagnostics Genetics Service, Sheffield Children's Hospital, Sheffield, UK

2014 - 2016: Lead Cancer Bioinformatician, Genomics England, London, UK

2013 – 2014: Bioinformatics Associate Research Scientist, Clinical Genome Project, St Jude Children's Research Hospital, Memphis TN, UK

2010 – 2013: Postdoctoral Scientist, The Pediatric Cancer Genome Project, St Jude Children's Research Hospital, Memphis TN, UK

2006 – 2010: PhD (Oncology) Institute of Cancer Research, London, UK

2002 – 2006: Bsc (With Honours in Experimental Pathology) University of Edinburgh, Scotland, UK

 
## Biography

I obtained my PhD in Molecular Biology/Cancer Biology at The Institute of Cancer Research in Colin Cooper's group where I developed a keen interest in the somatic changes which lead to the development of cancer. I pursued this interest at St Jude Children's Hospital in Memphis, TN, USA where I completed my postdoctoral training as part of the Pediatric Cancer Genome Project (PCGP) team. During this time I was responsible for the analysis of whole genome sequencing data in addition to transcriptomic data derived from childhood brain tumours. These analyses lead to discoveries which increased our understanding of these tumours, in particular, in Ependymoma I discovered a highly recurrent fusion gene, C11orf95-RELA.

I applied the skills I had learnt in bioinformatics and genome analysis to a clinical setting. I was lead analyst on the clinical proramme for genomics at St Jude, applying whole genome, exome and transcritome sequencing as a diagnostic tool. I took part in preparations to gain CAP-CLIA certification. On retrun to the UK I took a role as Lead Cancer Analyst on the 100,000 genomes project, applying my bioinformatics skills again in a clinically focussed setting. 

## Publications

For my updated and full list of publications, please see my ORCid entry:-

[https://orcid.org/0000-0002-8853-9435](https://orcid.org/0000-0002-8853-9435)

Selected Publications

1. Dunning, M.J, Vowler, S.L., Lalonde, E., Ross-Adams. H,. Boutros, P., Mills, I.G., Lynch, A.G, and Lamb A.D., “Mining Human Prostate Cancer Datasets: The camcAPP Shiny App“ EBio Medicine 17, (2017)

2. Ross-Adams, H., Lamb, A.D., Dunning, M.J., Halim, S., Lindberg, J., Massie, C.M., Egevad, L.A., Russell, R., Ramos-Montoya, A., Vowler, S.L., Sharma, N.L., Kay, J., Whitaker, H., Clark, J., Hurst, R., Gnanapragasam, V.J, Shah, N.C, Warren, A.Y., Cooper, C.S., Lynch, A.G., Stark, R., Mills, I.G., Gronberg, H. and Neal, D.E., “Integration of copy number and transcriptomics provides risk stratification in prostate cancer: A discovery and validation cohort study“ EBio Medicine 2(9):1133-1144, (2015).

3. Weaver, J.M.J., Shannon, N., Lynch, A.G., Ross-Innes, C.S., Forshew, T., Barbera, M., Ong, C.J., Lao-Sirieix, P., Dunning, M.J., Smith, L., Smith, M., Carvalho, B., O’Donovan. M., Underwood, T., Murtaza, M., May, A.P., Grehan, N., Hardwick, R., Davies, J., Olomi, A., Aparicio, S., Rozenfeld, N., Eldridge, M., Caldas, C., Edwards, P.A.W., Tavare, S. and Fitzgerald, R.C. “Ordering of mutations in preinvasive disease stages of esophageal carcinogenesis” Nature Genetics epub ahead of print (2014) ?

4. Dawson, S.J., Tsui, D.W.Y, Murtaza, M., Biggs, H., Rueda, O.M., Chin, S.F., Dunning, M.J., Gale, D., Forshew, T., Mahler-Araujo, B., Rajan, S., Humphray, S., Becq, J., Halsall, D., Wallis, M., Bentley, D., Caldas, C. and Rosenfeld, N., “Analysis of Circulating Tumour DNA to Monitor Metastatic Breast Cancer” The New England Journal of Medicine 368:1199-1209, 2013 ?

5. Curtis, C., Shah S., Chin, S.F., Turashvili, G., Rueda, O.M., Dunning, M.J., Speed, D., Lynch, A.G., Samarajiwa, S., Y, Yuan., METABRIC Group et al “The genomic and transcriptomic architecture of 2,000 breast tumours reveals novel subgroups.” Nature, 481(7381):389-93,2012 ?

6. Lynch, A.G., Chin, S.F., Dunning, M.J., Caldas, C., Tavare, S., and Curtis, C., “Calling sample mix-ups in cancer population studies” PLoS One, 7(8) e41815, 2012 ?

7. Ritchie, M.E., Dunning, M.J., Smith, M.L., Shi, W, and Lynch, A.G., “BeadArray expression analysis using bioconductor” PLoS Computational Biology 7(12) e1002276, 2011 ?

8. Dunning, M.J., Curtis, C., Barbosa-Morais, N.L., Caldas, C., Tavare, S. and Lynch, A.G., “The importance platform annotation in interpreting microarray data” Lancet Oncology 11(8):717, 2010 ?

9. Barbosa-Morais, N.L, Dunning, M.J., Samarajiwa, S.A., Darot, J.F.J., Ritchie, M.E., Lynch, A.G. and TavarE, S. “A re-annotation pipeline for Illumina BeadArrays: improving the interpretation of gene expression data” Nucleic Acids Research 38(3):e17, 2009 ?

10. Dunning, M.J., Barbosa-Morais, N.L, Lynch,. A.G., Tavare, S. and Ritchie, M.E. “Statistical issues in the analysis of Illumina data” BMC Bioinformatics 9:85, 2008 ?

11. Dunning, M.J. Smith, M.L., Ritchie, M.E. and Tavare, S. “beadarray: R classes and methods for Illumina bead-based data” Bioinformatics 23(16):2183-4 
