---
layout: page
organizer: Sheffield Bioinformatics Core
title: Introduction to RNA-seq analysis in R
excerpt: "RNA-seq in R"
startDate: 2018-07-09
endDate: 2018-07-10
startTime: 9.30
endTime: 17.00
city: sheffield
postcode: S1 3JD
country: United Kingdom
venue: Pam Liversidge Design Studio 1 - D06
description: "In this workshop, you will be learning how to analyse RNA-seq count data, using R. This will include reading the data into R, quality control and performing differential expression analysis and gene set testing, with a focus on the edgeR analysis workflow. You will learn how to generate common plots for analysis and visualisation of gene expression data, such as boxplots and heatmaps. You will also be learning how alignment and counting of raw RNA-seq data can be performed in R. This workshop is aimed at biologists interested in learning how to perform differential expression analysis of RNA-seq data when reference genomes are available.."
keywords: bioinformatics, NGS, RNA-seq, R
difficulty: beginner
contact: bioinformatics-core@sheffield.ac.uk
---

- Sheffield - 9th /10th July 2018
- 09:30am - 5pm
- Pam Liversidge Design Studio 1 - D06

<iframe src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d9519.181464571486!2d-1.4777067!3d53.3827108!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0x60e5580cdf19b137!2sPam+Liversidge+Building!5e0!3m2!1sen!2suk!4v1510862811609" width="600" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>

## Overview

In this workshop, you will be learning how to analyse RNA-seq count data, using R. This will include reading the data into R, quality control and performing differential expression analysis and gene set testing, with a focus on the DESEq2 analysis workflow. You will learn how to generate common plots for analysis and visualisation of gene expression data, such as boxplots and heatmaps. You will also be learning how alignment and counting of raw RNA-seq data can be performed in R. This workshop is aimed at biologists interested in learning how to perform differential expression analysis of RNA-seq data when reference genomes are available.

The first day of the workshop will include an introduction to the R language.

## Who should attend this course?

Researchers in life sciences who want to get an appreciation for the computational steps involved in RNA-seq analysis, but not neccesarily wishing to execute the pipeline for themselves. 

## Objectives:- After this course you should be able to:

- Design properly your RNA-Seq experiments considering advantages and limitations of RNA-seq assays
- Assess the quality of your datasets
- Know what tools are available in Bioconductor for RNA-seq data analysis and understand the basic object-types that are utilised
- Produce a list of differentially expressed genes from an RNA-seq experiment 

## Aims:- During this course you will learn about:

- RNA sequencing technology and considerations on experiment design
- Sources of variation in RNA-seq data
- Differential expression analysis using edgeR and DEseq2
- Annotation resources in Bioconductor
- Identifying over-represented gene sets among a list of differentially expressed genes 
    


## Software installation


## Instructors

- Mark Dunning, Bioinformatics Core Director
- To be confirmed

## Course Data

## Timetable (provisional)

Day 1
- 09:30 - 12:30 Introduction to R
- 12:30 - 13:30 LUNCH
- 13:30 - 14:30 Introduction to R (continued)
- 14:30 - 15:30 Introduction to RNA-seq and experimental design
- 15:30 - 17:00 Alignment and counting

Day 2

- 09:30 - 11:00 Quality assessment and pre-processing for RNA-seq
- 11:00 - 12:30 Differential Expression
- 12:30 - 13:30 LUNCH
- 13:30 - 15:30 Annotation and Visualisation of RNA-seq results
- 15:30 - 17:00 Gene set testing


## Registration 

[Click here](https://onlineshop.shef.ac.uk/conferences-and-events/faculty-of-medicine-dentistry-and-health/neuroscience/introduction-to-analysing-rnaseq-data-in-r) to book a place on the course

