---
title: "RNA-seq Analysis"
author: "Kat"
date: "13 February 2020"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

Notes here
```{r}
#read in metadata
sampleinfo <- read.delim("meta_data/sampleInfo.txt")
sampleinfo
rownames(sampleinfo) <- sampleinfo$run
```


```{r}
library(tximport)

```

```{r}
dirs <- list.files("salmon_quant/")
dirs

quant_files <- list.files("salmon_quant/", pattern = "quant.sf.gz", recursive = TRUE, full.names = TRUE)
names(quant_files) <- dirs
quant_files
```

```{r}
quant_files[1]
quants <- read.delim(quant_files[1])
head(quants)
```

```{r eval=FALSE, include=FALSE}
link_gtf_file <- "ftp://ftp.ensembl.org/pub/release-99/gtf/mus_musculus/Mus_musculus.GRCm38.99.chr.gtf.gz"

gtf_file <- "Mus_musculus.GRCm38.99.chr.gtf.gz"

download.file(link_gtf_file, destfile = gtf_file)
```
```{r}
gtf_file <- "Mus_musculus.GRCm38.99.chr.gtf.gz"

library(GenomicFeatures)
txdb <- makeTxDbFromGFF(gtf_file)

keytypes(txdb)
columns(txdb)
```

```{r}
k <- keys(txdb, keytype="TXNAME")
head(k)
tx_map <- select(txdb, keys = k, columns="GENEID", keytype="TXNAME")
head(tx_map)
```

```{r}
tx2gene <- tx_map
write.csv(tx2gene, file="tx2gene.csv", row.names = FALSE, quote = FALSE)
```

```{r}
txi <- tximport(quant_files, type="salmon", tx2gene = tx2gene, ignoreTxVersion = TRUE)

names(txi)
head(txi$counts)
```

```{r}
library(DESeq2)

dds <- DESeqDataSetFromTximport(txi, colData = sampleinfo, design = ~CellType)

head(dds)
```

```{r}
sum(assay(dds)[,1])
colSums(assay(dds))

barplot(colSums(assay(dds)))
abline(h=20e6)
```

```{r}
boxplot(log10(assay(dds)))
```


```{r}
vsd <- vst(dds)
boxplot(assay(vsd))
```

```{r}
sampleDists <- dist(t(assay(vsd)))

library(RColorBrewer)
library(pheatmap)

sampleDistsMatrix <- as.matrix(sampleDists)
rownames(sampleDistsMatrix) <- paste(colData(dds)$CellType, colData(dds)$Status, sep = "-")
colnames(sampleDistsMatrix) <- colData(dds)$Name

colors <- colorRampPalette(rev(brewer.pal(9, "Blues")))(255)

pheatmap(sampleDistsMatrix, 
         col=colors)
```

```{r}
plotPCA(vsd, intgroup = "CellType")
```

```{r}
library(stringr)
library(dplyr)
sampleinfo_corrected <- sampleinfo
sampleinfo_corrected <- mutate(sampleinfo_corrected, CellType = str_to_lower(CellType))
sampleinfo_corrected <- mutate(sampleinfo_corrected, Status = str_trim(Status))
sampleinfo_corrected <- mutate(sampleinfo_corrected, CellType = ifelse(Name == "MCL1-DH", "basal", CellType) )
sampleinfo_corrected <- mutate(sampleinfo_corrected, CellType = ifelse(Name == "MCL1-LA", "luminal", CellType))

write.table(sampleinfo_corrected, file="meta_data/sampleInfo_corrected.txt", sep="\t", row.names = F)
```

```{r}
dds <- DESeqDataSetFromTximport(txi, colData = sampleinfo_corrected, design = ~CellType)
vsd <- vst(dds)
plotPCA(vsd, intgroup = "CellType")

#plot pca for CellType and Status 
plotPCA(vsd, intgroup = c("CellType", "Status"))

plot_data <- plotPCA(vsd, intgroup = c("CellType", "Status"), returnData=TRUE)
head(plot_data)

library(ggplot2)
ggplot(plot_data, aes(x = PC1, y = PC2, col = CellType, pch=Status)) + geom_point(size=5)
```

```{r}
library(DESeq2)
library(tximport)

dirs <- list.files(file.path("salmon_quant/"))
quant_files <- paste0("salmon_quant/", dirs, "/quant.sf.gz")
quant_files

tx2gene <- read.csv("tx2gene.csv")

txi <- tximport(quant_files, type="salmon", tx2gene = tx2gene, ignoreTxVersion = TRUE)

sampleinfo <- read.delim("meta_data/sampleInfo_corrected.txt")
rownames(sampleinfo) <- sampleinfo$run

dds <- DESeqDataSetFromTximport(txi,
                                colData = sampleinfo,
                                design = ~CellType)
```

```{r}
colData(dds)

design(dds) <- ~CellType
de.cellType <- DESeq(dds)
```
```{r}
results(de.cellType)

library(dplyr)
library(tibble)

results.cellType <- as.data.frame(results(de.cellType)) %>%
  rownames_to_column("GeneID")

results.cellType
```

```{r}
arrange(results.cellType, padj) %>%
  head(n=10)

arrange(results.cellType, padj) %>%
  write.csv("basal_vs_luminal_DESeq_all.csv")

arrange(results.cellType, padj) %>%
  filter(padj < 0.05) %>%
  write.csv("basal_vs_luminal_DESeq_DE.csv")
```


```{r}
dds.status <- dds
design(dds.status) <- ~Status

de.status <- DESeq(dds.status)
results(de.status)

results.status <- as.data.frame(results(de.status)) %>%
  rownames_to_column("GeneID")

head(results.status)

arrange(results.status, padj) %>%
  filter(padj < 0.05, abs(log2FoldChange) > 1 ) %>%
  write.csv("virgin_vs_lactation_DESeq_DE_padj0.05_abslogFC1.csv")
```

```{r}

results(de.cellType, contrast = c("CellType", "luminal", "basal"))

results(de.cellType, contrast = c("CellType", "basal", "luminal"))

results(de.status, contrast = c("Status", "pregnancy", "lactation"))
results(de.status, contrast = c("Status", "pregnancy", "virgin"))
resultsNames(de.status)

```

```{r}
venn_data <- data.frame(CellType = results.cellType$padj < 0.05, 
                        Status = results.status$padj < 0.05)

library(limma)
vennDiagram(venn_data)
```


```{r}
dds.mf <- dds
design(dds.mf) <- ~CellType+Status
de.mf <- DESeq(dds.mf)
```

```{r}
results.mf <- results(de.mf, contrast = c("Status", "lactation", "virgin"))
resultsNames(de.mf)
```

```{r}
dds <- estimateSizeFactors(dds)
countMatrix <- counts(dds, normalized = TRUE)
head(countMatrix)

write.csv(countMatrix, file="normalized_counts.csv")

save(de.cellType, de.status, de.mf, file="Robjects/DE.RData")
load("Robjects/DE.RData")
```

```{r}
results.mf
```

```{r}
DESeq2::plotMA(results.mf)
```

```{r}
results.ordered <- data.frame(results.mf) %>% 
  rownames_to_column("ENSEMBL") %>% 
  arrange(padj)
results.ordered
```

```{r}
ggplot(results.ordered, aes(x = log2FoldChange, -log10(padj))) + geom_point()
```

```{r}
plotCounts(dds, "ENSMUSG00000000381", intgroup=c("Status"))
```

```{r}
count_data <-plotCounts(dds, "ENSMUSG00000000381",intgroup = c("Status","CellType"),returnData = TRUE)
count_data
ggplot(count_data, aes(x=Status,y=count,col=Status)) + geom_point()
```

```{r}
ggplot(count_data, aes(x = Status, y = log10(count),col=Status)) + geom_jitter(width=0.1) + facet_wrap(~CellType)
```

```{r}
results.ordered
assay(vsd)[c("ENSMUSG00000000381","ENSMUSG00000061937"),]
top_genes <- results.ordered$ENSEMBL[1:10]
top_genes
pheatmap(assay(vsd)[top_genes, ])
```

```{r}
sampleInfo <- data.frame(colData(dds))[,c("CellType","Status")]
pheatmap(assay(vsd)[top_genes,],
         annotation_col = sampleInfo)
```

```{r}
library(biomaRt)
listMarts()
```

```{r}
ensembl <- useMart("ENSEMBL_MART_ENSEMBL")
#listDatasets(ensembl) %>% filter(grepl("Human",description))
listDatasets(ensembl) %>% filter(grepl("Mouse",description))
ensembl <- useMart("ENSEMBL_MART_ENSEMBL",dataset = "mmusculus_gene_ensembl")
```

```{r}
listFilters(ensembl)
```

```{r}
listAttributes(ensembl)
```

```{r}
getBM(attributes = c("external_gene_name","entrezgene_id","ensembl_gene_id","chromosome_name"),
      filters = "ensembl_gene_id",
      values = top_genes,
      mart = ensembl)
```

```{r}
anno <- getBM(attributes = c("external_gene_name","entrezgene_id","ensembl_gene_id"),
      filters = "ensembl_gene_id",
      values = results.ordered$ENSEMBL,
      mart = ensembl)
head(anno)
anno <- rename(anno, ENSEMBL = "ensembl_gene_id")
```

```{r}
results.annotated <- left_join(results.ordered, anno)
head(results.annotated)
```

```{r}
N <- 20

top_genes <- results.annotated$ENSEMBL[1:N]
gene_names <- results.annotated$external_gene_name[1:N]

pheatmap(assay(vsd)[top_genes,],
         annotation_col = sampleInfo,
         labels_row = gene_names)
```

```{r}
ggplot(results.annotated, aes(x = log2FoldChange, y = -log10(padj),label=external_gene_name)) + geom_point() + geom_text()
```

```{r}
N <- 50
top_genes <- results.annotated$ENSEMBL[1:N]

results.annotated %>% 
  mutate(Label = ifelse(ENSEMBL %in% top_genes, external_gene_name,"")) %>% 
  ggplot(aes(x = log2FoldChange, y = -log10(padj), label=Label)) + geom_point() + geom_text()
```

```{r}
library(ggrepel)
results.annotated %>% 
  mutate(Label = ifelse(ENSEMBL %in% top_genes, external_gene_name,"")) %>% 
  ggplot(aes(x = log2FoldChange, y = -log10(padj), label=Label)) + geom_point() + geom_text_repel()
```

```{r}
filter(results.annotated, padj < 0.05) %>% 
  pull(ENSEMBL) %>% write.table("sig_genes.txt",row.names=FALSE, quote=FALSE)
```

```{r}
results.annotated %>% 
  pull(ENSEMBL) %>% 
  write.table("gene_universe.txt",row.names=FALSE,quote=FALSE)
```

```{r}
library(clusterProfiler)
```

```{r}
library(org.Mm.eg.db)
sig_genes <- filter(results.annotated, padj < 0.05) %>% 
  pull(ENSEMBL)

universe <- results.annotated %>% 
  pull(ENSEMBL) %>% 

enrich_go <- enrichGO(gene = sig_genes,
                      OrgDb = org.Mm.eg.db,
                      keyType = "ENSEMBL",
                      universe = universe)
```

```{r}
dotplot(enrich_go)
```

```{r}
emapplot(enrich_go)
```

```{r}
sig_genes_entrez <- filter(results.annotated, padj < 0.05) %>% 
  pull(entrezgene_id)
keg_res <- enrichKEGG(gene=sig_genes_entrez, organism="mmu")
head(keg_res)
```

```{r}
library(pathview)
logFC <- results.annotated$log2FoldChange
names(logFC) <- results.annotated$entrezgene_id

pathview(gene.data = logFC,
         pathway.id = "mmu03010",
         species = "mmu",limit=list(gene=5,cpd=1))
browseKEGG(keg_res, "mmu03010")
```

